package.json

```diff
@@ -57,8 +57,8 @@
   "devDependencies": {
     "@actions/core": "^1.0.0",
     "@actions/github": "^2.2.0",
-    "@oclif/dev-cli": "^1.22.2",
```

Comment:
Was there any testing done to validate that dev-cli cmds still work via oclif/core ?

Reply:
yes, ran the command oclif-dev readme was executed, but didnt work.
The output was zsh: command not found: oclif-dev